### PR TITLE
Add Yle

### PR DESCRIPTION
**Pull request by slothtown**

### CODE REVIEW

**Comment by Carlgo11** [comment]
I don't see any places to log in :thinking:

**Comment by slothtown** [comment]
There should be a login icon on the top right of the page. Maybe it's easier to find on the English page: https://yle.fi/news

File: entries/y/yle.fi.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Yle": {
+    "domain": "yle.fi",
+    "contact": {
+      "facebook": "yleisradio",
+      "twitter": "yleuutiset",
+      "form": "https://asiakaspalvelu.yle.fi/csp",
```
Comment (Carlgo11) [inline entries/y/yle.fi.json]:
```suggestion
      "email": "yleinfo@yle.fi",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/y/yle.fi.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "Yle": {
+    "domain": "yle.fi",
+    "contact": {
+      "facebook": "yleisradio",
+      "twitter": "yleuutiset",
```
Comment (Carlgo11) [inline entries/y/yle.fi.json]:
```suggestion
      "twitter": "Yleisradio",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**